docs/manual/developer_guide.adoc

```diff
@@ -103,16 +104,16 @@ There is also an official https://fedoraproject.org/wiki/How_to_debug_installati
 
 Following packages are needed to run unit tests:
 
-```
+----
 anaconda
 openscap-python
 python-cpio
 python-mock
-python-nose
-```
+python-pytest
```

Comment:
IMO we should explicitly state `python3` for these packages. There is also `openscap-python3`.

Reply:
I have explicitly specified `python3-`, but `pycurl` is missing for both branches, so I consider it out of scope of this PR. See #78.